### ISSUE TITLE
Tracks and additional details for Music Albums

### ISSUE THREAD

**Issue by rokville** [mProjectsCode/obsidian-media-db-plugin#30]
There is a lot of extra info that would be nice to have available for music albums
e.g., 
Track info: track number, song name, track length, additional artists
[example with normal track details](https://musicbrainz.org/release/a2398db7-52ca-4924-9ef9-dcfc0f59d95b)
Extended track credits (too much, probably but cool if available)
[example with LOTS of additional details](https://musicbrainz.org/release/51eb10e0-2b9e-3906-a8f1-db74b346cba5)

General Album info
cover art, producer(s), length of album, number of tracks, label
[see right side of the example](https://musicbrainz.org/release/51eb10e0-2b9e-3906-a8f1-db74b346cba5)

Thanks, this is awesome!

**Comment by mProjectsCode** [comment]
Yes, the plugin currently only queries so called music release groups.
Getting more info requires selecting a release from the release group. For that I need to finish the search and select modal rework, which I am working on currently for the bulk import feature.

**Comment by rokville** [comment]
Thank you for your efforts and the update!

**Comment by AkhirAlibhai** [comment]
@mProjectsCode 
I'm interested in picking up this issue, have you started working on it? Would love to see this added!

**Comment by mProjectsCode** [comment]
No i haven't, feel free to do so

**Comment by Kullenej** [comment]
> I'm interested in picking up this issue, have you started working on it? Would love to see this added!

Have you started on this? Would love to have album cover in the template! 😄

**Comment by ProudBenzene** [comment]
The same needs! I hope to collect more meta information about albums through Media DB plugin, such as list of songs in album, number of tracks, composer, lyricist, music duration, total album duration, etc. MusicBrainz is full of data about music meta information. I hope you can optimize it more!🌹

**Comment by ProudBenzene** [comment]
This is a sample of album collection notes I made, which shows the music-related metadata currently needed by music lovers like me for your reference!🤗

![Image](https://github.com/user-attachments/assets/279a39a0-0054-4965-b08f-096b2bb2633d)
